Source/GmmLib/CMakeLists.txt

```diff
@@ -98,24 +98,6 @@ macro(GmmLibSetTargetConfig libTarget)
 
 endmacro()
 
-set (CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} /W1 /WX-")
-set (CMAKE_EXE_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} /W1 /WX-")
-
-# GMMLIBsetLinkerFlags: apply linker flags for given configuration
```

Comment:
Cannot remove this macro, this is being used to set linker settings in Linker.cmake..
Rest of the changes are fine and under testing right now..

Reply:
I don't see that this macro is used anywhere:
```
# fgrep -rsn GMMLIBsetLinkerFlags ..
../Source/GmmLib/CMakeLists.txt:104:# GMMLIBsetLinkerFlags: apply linker flags for given configuration
../Source/GmmLib/CMakeLists.txt:107:macro (GMMLIBsetLinkerFlags linkerFlags linkerTarget)
```

That's why I have deleted it. Why do you need it?

Reply:
aah! I checked..Looks like it has been removed from [Linker.cmake](https://github.com/intel/gmmlib/blob/master/Source/GmmLib/Linux.cmake).. Earlier we used to call from there to set linker flags property. It should be safe to remove then.